services/ml/compilation_impl.cc

```diff
@@ -58,7 +58,7 @@ void CompilationImpl::Finish(int32_t preference, FinishCallback callback) {
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
   if (command_line->HasSwitch(switches::kUseInferenceEngine)) {
 #if defined(OS_LINUX) || defined(OS_WIN)
-    delegate_ = std::make_unique<CompilationDelegateIe>(this);
+    // delegate_ = std::make_unique<CompilationDelegateIe>(this);
```

Comment:
remove this one?

Reply:
All of relative code has been removed with latest commit. PTAL, thanks.